setup.scripts/mailwatch/mailwatch.inc

```diff
@@ -274,9 +274,9 @@ function install-sudo() {
     else
         logprint "Not installing sudo."
     fi
-    if ! ( type "sudo" > /dev/null 2>&1 ) ; then
+    if ( type "sudo" > /dev/null 2>&1 ) ; then
         logprint "Sudo is installed so we are configuring it for use with MailWatch"
-        sed -i -e "s/User_Alias MAILSCANNER = .*/User_Alias MAILSCANNER = $Webuser" "$MailWatchTmpDir/tools/sudo/mailwatch"
+        sed -i -e "s/User_Alias MAILSCANNER = .*/User_Alias MAILSCANNER = $Webuser/" "$MailWatchTmpDir/tools/sudo/mailwatch"
```

Comment:
Did you change the setup depending of the MTA?

```
# If you use Exim enable these
#Cmnd_Alias EXIM_QUEUE_IN = /usr/sbin/exim -bpc
#Cmnd_Alias EXIM_QUEUE_OUT = /usr/sbin/exim -bpc -DOUTGOING
#Cmnd_Alias SENDMAIL_QUEUE_IN = /usr/bin/mailq -bp -OQueueDirectory=/var/spool/mqueue.in
#Cmnd_Alias SENDMAIL_QUEUE_OUT = /usr/bin/mailq -bp
#MAILSCANNER ALL= NOPASSWD: MAILSCANLINT, SPAMASSASSINLINT, EXIM_QUEUE_IN, EXIM_QUEUE_OUT, SENDMAIL_QUEUE_IN, SENDMAIL_QUEUE_OUT

# If you use postfix enable these
#Cmnd_Alias POSTFIX_QUEUE = /usr/sbin/postqueue
#MAILSCANNER ALL= NOPASSWD: MAILSCANLINT, SPAMASSASSINLINT, POSTFIX_QUEUE

```

Reply:
Not yet

Reply:
fixed in https://github.com/mailwatch/mailwatch-install-script/pull/38/commits/59e0d3f8f31f75a75d960094c7965db8a6593bbd